l10n/en/firefox/sync.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the { -brand-name-mozilla } Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/sync/
+
+firefox-sync = { -brand-name-firefox-sync }
```

Comment:
Fluent IDs should use a standard prefix that associates them with the page where they are being used. This helps avoid naming collisions. See: https://bedrock.readthedocs.io/en/latest/l10n.html#ftl-string-ids

Reply:
I also realize now that we missed this with the /quantum page :(

Reply:
PR to update firefox/quantum: https://github.com/mozilla/bedrock/pull/9623/files
thanks Alex :)